tests/prague/eip7692_eof_v1/eip7069_extcall/test_gas.py

```diff
@@ -97,22 +128,46 @@ def gas_test(
             storage={
                 slot_warm_gas: warm_gas,
                 slot_cold_gas: cold_gas,
+                slot_oog_call_result: value_call_legacy_abort,
+                slot_sanity_call_result: value_call_legacy_success,
             },
         ),
     }
 
-    tx = Transaction(to=address_legacy_harness, gas_limit=2_000_000, sender=sender)
+    tx = Transaction(to=address_legacy_harness, gas_limit=20_000_000, sender=sender)
 
     state_test(env=env, pre=pre, tx=tx, post=post)
 
 
 @pytest.mark.parametrize(
-    ["opcode", "pre_setup", "cold_gas", "warm_gas"],
+    ["opcode", "pre_setup", "cold_gas", "warm_gas", "new_account"],
     [
-        pytest.param(Op.EXTCALL, Op.PUSH0, 2600, 100, id="EXTCALL"),
-        pytest.param(Op.EXTCALL, Op.PUSH1(1), 2600 + 9000, 100 + 9000, id="EXTCALL_with_value"),
-        pytest.param(Op.EXTDELEGATECALL, Op.NOOP, 2600, 100, id="EXTSTATICCALL"),
-        pytest.param(Op.EXTSTATICCALL, Op.NOOP, 2600, 100, id="EXTDELEGATECALL"),
+        pytest.param(Op.EXTCALL, Op.PUSH0, 2600, 100, False, id="EXTCALL"),
+        pytest.param(
+            Op.EXTCALL, Op.PUSH1(1), 2600 + 9000, 100 + 9000, False, id="EXTCALL_with_value"
```

Comment:
```suggestion
            Op.EXTCALL, Op.PUSH1(1), COLD_ACCOUNT_ACCESS_GAS + CALL_WITH_VALUE_GAS, WARM_ACCOUNT_ACCESS_GAS + CALL_WITH_VALUE_GAS, False, id="EXTCALL_with_value"
```
It would be nice if these values were spelt out in the form of named constants. It just makes it easier to follow what the test is trying to test exactly.